protocol/database-scripting/README.md

```diff
@@ -0,0 +1,121 @@
+Database Scripting
+==============
+
+A protocol for running console scripts on the Healthify app's production database.
+
+Development
+-----------
+
+Develop the scripting story similarly to any other story. In this case, this would
+entail running your script at the rails console of your local server during development.
+
+Typically, our scripts should live in the `db/data_migrations/` subdirectory within
+the `healthify/` app repo. In rare cases where that's not feasible, you might use
+a secret [gist](https://help.github.com/articles/about-gists/), whose you link you
+should post on Pivotal Tracker and share with your Code Reviewers.
+
+While some scripts only lend themselves to being performed via pure SQL, where it
+is possible, you should leverage ActiveRecord to perform our `INSERT`/`UPDATE`/`DELETE`
+operations so that we leverage ActiveRecord validations and callbacks.
+
+While developing the script, it is critical to plan our remediation strategy if
+something does go awry.
+
+One useful approach to consider is to wrap the `INSERT`/`UPDATE`/`DELETE` commands within
+a transaction, and then to assert on certain states of the database after the script
+is complete, rolling back all changes if the assertions fail.
+
+Further, it's important to be able to execute a complete reversal of your changes
+in the event that there are unexpected failures. Ideally, we would write the commands
+we would need to immediately reverse our changes if necessary (i.e. a reversal script).
+
+For `INSERT` operations, we would need to retain data about what records ended up
+being `INSERT`-ed by our script, and then the reversal script would delete those
+records.
+
+For `UPDATE`/`DELETE` operations, we would want to dump the data of all the records
+we will be interacting with into a "rescue table" (a table of the data which we would keep
+for only around a week before deleting. Such a table would ideally live in a
+`rescue_table.*` DB schema). Our reversal script would then perform a COPY of the data
```

Comment:
We should use `rescue` instead of `rescue_table` as the name of the schema.

See https://github.com/healthify/data_scripting/pull/1. Tagging @switzersc